app/views/events/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<section class="event my-4">
+<h2>Belongs to: <%= @event.creator.name %></h2>
+<h5 class="text-primary"><span class="text-secondary">Event:</span>  <%= @event.location %> - <%= @event.date %></h5>
+<section class="attendants">
+  <h3 class="h4">People attending/attended this event:</h3>
+  <ul class="attendants-list">
+  <% @event.attendees.each do |attendee| %>
+    <li><%= attendee.name %></li>
+  <% end %>
```

Comment:
- [ ] There should absolutely be no logic in your views. Views are only for displaying information that became available to it from its corresponding controller. Kindly use this [resource](http://codefol.io/posts/Where-Do-I-Put-My-Code/) to learn more on this.

- Kindly use the rails helpers for your logic. Logic includes, Loops and If statements.